vllm/core/scheduler.py

```diff
@@ -4,8 +4,9 @@
 import time
 from collections import deque
 from dataclasses import dataclass, field
-from typing import (Callable, Deque, Dict, Iterable, List, Optional, Set,
-                    Tuple, Union)
+from typing import Callable, Deque, Dict, Iterable, List, Optional
+from typing import Sequence as SequenceType
```

Comment:
Elsewhere we use `GenericSequence` as the alias for this

Reply:
ah, sure enough, thanks.